pdf/model/appender.go

```diff
@@ -132,7 +141,7 @@ func (a *PdfAppender) addNewObjects(obj core.PdfObject) {
 	switch v := obj.(type) {
 	case *core.PdfIndirectObject:
 		// Check the object is changing.
-		// If the indirect object has not the readonly parser then the object is changed.
+		// If the indirect object has not the read-only parser then the object is changed.
```

Comment:
If the indirect object does not have the read-only parser, then the object has changed.